tests/test_celery.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+from merlin.celery import app
+
+
+def test_broker_url():
```

Comment:
This is not going to work in general, on LANL this would fail because it is not ampqs.

Reply:
You can also use redis for the broker.